misc/xml-to-record-converter/src/main/java/io/ballerina/xmltorecordconverter/XMLToRecordConverter.java

```diff
@@ -171,19 +174,25 @@ public static XMLToRecordResponse convert(String xmlValue, boolean isRecordTypeD
         return DiagnosticUtils.getDiagnosticResponse(diagnosticMessages, response);
     }
 
+    public static XMLToRecordResponse convert(String xmlValue, boolean isRecordTypeDesc, boolean isClosed,
+                                              boolean forceFormatRecordFields) {
```

Comment:
Shall we make it deprecated as we might no longer use this, WDYT?

Reply:
This is used by the old plugin versions. Hence let's not mark it as deprecated